tests/libnet/ipaddress.go

```diff
@@ -5,7 +5,15 @@ import (
 	netutils "k8s.io/utils/net"
 )
 
-func GetIp(ips []string, family v1.IPFamily) string {
+func GetPodIpByFamily(pod *v1.Pod, family v1.IPFamily) string {
```

Comment:
this change is not related to the commit, and is also refactored in following commits,
i really thing that we should take the time please, in the end of the PR at least, just before merge, to reorder and have cleaner, meaningful commits,

it would help to learn from the PRs in the future.